planner/core/logical_plan_builder.go

```diff
@@ -879,56 +879,60 @@ func (b *PlanBuilder) buildSort(p LogicalPlan, byItems []*ast.ByItem, aggMapper
 	return sort, nil
 }
 
-// getUintForLimitOffset gets uint64 value for limit/offset.
-// For ordinary statement, limit/offset should be uint64 constant value.
-// For prepared statement, limit/offset is string. We should convert it to uint64.
-func getUintForLimitOffset(ctx sessionctx.Context, n ast.Node) (uint64, error) {
+// getUintFromNode gets uint64 value from ast.Node.
+// For ordinary statement, node should be uint64 constant value.
+// For prepared statement, node is string. We should convert it to uint64.
+func getUintFromNode(ctx sessionctx.Context, n ast.Node) (uVal uint64, isNull bool, isExpectedType bool) {
 	var val interface{}
 	switch v := n.(type) {
 	case *driver.ValueExpr:
 		val = v.GetValue()
 	case *driver.ParamMarkerExpr:
 		param, err := expression.GetParamExpression(ctx, v)
 		if err != nil {
-			return 0, errors.Trace(err)
+			return 0, false, false
```

Comment:
Is it safe to ignore the error?

Reply:
In the original code https://github.com/pingcap/tidb/pull/9004/files#diff-b9e75a99170ea35a67074d5d3d3305adL925, it also ignores the error, so I think it is safe to ignore it.